deploy/cloudformation/create-stack.sh

```diff
@@ -3,6 +3,7 @@ STACK_NAME="${1:-$STACK_NAME}"
 FLEET_URL="${2:-$FLEET_URL}"
 ENROLLMENT_TOKEN="${3:-$ENROLLMENT_TOKEN}"
 ELASTIC_AGENT_VERSION="${4:-elastic-agent-8.8.0-SNAPSHOT-linux-arm64}"
+KEY_NAME="${5:-$KEY_NAME}"
```

Comment:
@amirbenun - I received the following error if the ssh key name is not provided.
```
An error occurred (ValidationError) when calling the CreateStack operation: Parameters: [KeyName] must have values
```

Reply:
Yes, we removed the key as part of [Blocking SSH connections on CloudFormation template](https://github.com/elastic/security-team/issues/6279).
Any chance you were using an old template?

Reply:
yes, reverted